scripts/projectView.js

```diff
@@ -5,7 +5,6 @@ projectsView.handleMainNav = function() {
     e.preventDefault();
     $('.page-content').hide();
     var $clickEvent = $(this);
-    console.log($clickEvent);
```

Comment:
be sure to remove your console.log statements statements when the functionality is complete. dry out the code as much as possible so readability isn't compromised.